harvard-lts/CURIOSity
# Contacts Display Issue

**Contacts Display Issue**
* * *

**JIRA Ticket**: [LTSMAINT-761](https://jira.huit.harvard.edu/browse/LTSMAINT-761)

# What does this Pull Request do?
Adds space below each contact entry on item detail page and updates styles of contact entry on about pages to match with item detail page.

# How should this be tested?
* Spin up Curiosity from this branch
* Sign in, navigate to the demo site, and create two contacts (Exhibit Dashboard --> About pages --> Add contact)
* Select both contacts to be added to pages (and make sure you click the "Save changes" button)
* Go to an item detail page in the demo site: you should see two contacts in the right sidebar and there should be enough space between them so that you can tell they are two separate contacts
* Go to any about page: contact information should be styled to match what you see on the item detail page (Trueno font, smaller font size, enough space between contacts if multiple are present)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@VanessaVenti @enriquediaz
